src/Chart/config.jsx

```diff
@@ -34,22 +34,28 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import { connect } from 'react-redux';
-import ChartView from './ChartView';
-
-const yRange = {
-    min: -110,
-    max: -20,
+export const yRange = {
+    min: 110,
+    max: 20,
 };
 
-const adjustY = y => -(yRange.min + yRange.max) - y;
+// Official Nordic colors, taken from https://github.com/NordicSemiconductor/pc-nrfconnect-shared/blob/9bb1f72849/src/variables.scss
```

Comment:
Would be nice that in the future we could just import the colors instead of defining new ones.

Reply:
I think it is a good idea to also place them in `shared`, so that we do not have two places far apart from each other where these are defined. I will create a PR for that.